package.json

```diff
@@ -12,8 +11,13 @@
     "test": "karma start test/karma.conf.js"
   },
   "window": {
-    "show": false,
-    "icon": "images/bf_icon_128.png"
+    "icon": "images/bf_icon_128.png",
+    "id": "main-window",
+    "frame": "chrome",
+    "innerBounds": {
+        "minWidth": 1024,
+        "minHeight": 550
+    }
```

Comment:
@McGiverGim This innerBounds section not being effective is what I'm talking about with the min_width/min_height suggestion.

Reply:
Maybe we must try to open the window from JavaScript and pass them as parameters, similar to the Chrome version.
My PR opened it from JavaScript but I have not passed parameters.